pulsar-broker/src/main/java/org/apache/pulsar/PulsarBrokerStarter.java

```diff
@@ -273,8 +277,9 @@ public void shutdown() {
 
 
     public static void main(String[] args) throws Exception {
+        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd hh:mm:ss,SSS");
         Thread.setDefaultUncaughtExceptionHandler((thread, exception) -> {
-            log.error("Uncaught exception in thread {}: {}", thread.getName(), exception.getMessage(), exception);
+            System.out.println(String.format("%s [%s] error Uncaught exception in thread %s: %s", dateFormat.format(new Date()), thread.getContextClassLoader(), thread.getName(), exception.getMessage()));
```

Comment:
@merlimat  @foreversunyao use log4j properties  redirect to System.out  would be better
like 
```yaml
- name: log4j.appender.CONSOLE
  default: org.apache.log4j.ConsoleAppender
- name: log4j.appender.CONSOLE.Threshold
  default: ERROR
- name: log4j.appender.CONSOLE.Target
  default: System.out
```